src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -16,6 +17,11 @@ namespace Microsoft.Build.BackEnd
     /// </summary>
     internal static class ItemGroupLoggingHelper
     {
+        /// <summary>
+        /// The default character limit for logging parameters.
```

Comment:
Might want to explain why. I think this was arbitrary, but that would be my first question.

Reply:
The beauty is that it is arbitrary 😁. I'll link to the original issue for some context.

> We spitballed a limit like "the first 10k characters of the item inputs as text".